util/slice/slice_test.go

```diff
@@ -41,10 +34,13 @@ func (s *testSliceSuite) Test(c *C) {
 		{[]int{2, 2, 4}, true, false, true},
 	}
 
-	for _, t := range tests {
-		even := func(i int) bool { return t.a[i]%2 == 0 }
-		c.Assert(AnyOf(t.a, even), Equals, t.anyOf)
-		c.Assert(NoneOf(t.a, even), Equals, t.noneOf)
-		c.Assert(AllOf(t.a, even), Equals, t.allOf)
+	for _, test := range tests {
+		t.Run(strings.Join(strings.Fields(fmt.Sprint(test.a)), ","), func(t *testing.T) {
```

Comment:
Maybe `fmt.Sprint(test.a)` is enough. Or just `fmt.Sprintf("%d: %s", idx, test.a)`, where idx is `for idx, test := range tests {`.

Reply:
Thanks for your advice. Will fix this.